src/_repobee/ext/gitlab.py

```diff
@@ -338,19 +330,15 @@ def get_repo_urls(
     ) -> List[str]:
         """See :py:meth:`repobee_plug.API.get_repo_urls`."""
         group_name = org_name if org_name else self._group_name
-        group_url = "{}/{}".format(self._base_url, group_name)
+        group_url = f"{self._base_url}/{group_name}"
         repo_urls = (
             [
-                "{}/{}.git".format(group_url, repo_name)
+                f"{group_url}/{repo_name}.git"
                 for repo_name in master_repo_names
             ]
             if not teams
             else [
-                "{}/{}/{}.git".format(
-                    group_url,
-                    team,
-                    plug.generate_repo_name(str(team), master_repo_name),
-                )
+                f"{group_url}/{team}/{plug.generate_repo_name(str(team), master_repo_name)}.git"
```

Comment:
This line is quite a bit too long.

```suggestion
                f"{group_url}/{team}/
                f"{plug.generate_repo_name(str(team), master_repo_name)}.git"
```